kalite/updates/api_views.py

```diff
@@ -129,10 +130,12 @@ def start_video_download(request):
     lang = json.loads(request.body or "{}").get("lang", "en")
 
     youtube_ids = get_download_youtube_ids(paths, language=lang)
-
+    ids_to_queue = {id_: title for (id_, title) in youtube_ids.iteritems()
```

Comment:
How can I ensure that a file isn't downloaded *again* if it's already on disk? Is this information contained somehow in `extra_fields`? Or can I infer it from `files_complete`? What is the significance of that field?